### PR TITLE
fix: unsupported push notifications

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Skip the step when PN is not supported.

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://fix-pn-onboarding.preview.app.daily.dev

### CODE REVIEW

**Review by capJavert — APPROVED** [review]

File: packages/webapp/pages/onboarding.tsx [hunk 1]
```diff
@@ -187,7 +189,11 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.ContentTypes);
     }
 
-    if (activeScreen === OnboardingStep.ContentTypes && isMobile) {
+    if (
+      activeScreen === OnboardingStep.ContentTypes &&
+      isMobile &&
+      isPushSupported
```
Comment (rebelchris) [inline packages/webapp/pages/onboarding.tsx:195]: We need to either fix the export of `isPushSupported` or do this:
https://github.com/dailydotdev/apps/commit/15eed06762d267a923b56637b1ae73a192102666

(Best is first advice)

Just try it in Brave browser it blocks OneSignal, so you can see if it works or not.

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/webapp/pages/onboarding.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/pages/onboarding.tsx:195]: I will fix it from the root to avoid any issues down the line 👀

**Review by rebelchris — APPROVED** [review]
Thanks the other hotfix i did still works i assume right? 👍

**Comment by sshanzel** [comment]
> Thanks the other hotfix i did still works i assume right? 👍

Worked both locally, and prod on the first fix. Will check how it works with the update on is push supported.

**Comment by sshanzel** [comment]
Yeah, the item is still not found in the checklist.

IMG_0114

**Pull request merged**